### PR TITLE
new proposals Aug3 - Aug13

### PR DESCRIPTION
**Pull request by pmoosh**

### CODE REVIEW

File: site/content/events/2015-siliconvalley/proposals/Application Driven Infrastructure/index.txt [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+talk: true
+ignite: false
+selected: false
+author: Speaker 34
+title: "Application Driven Infrastructure"
+---
+
+**Abstract:**
+
+GOn the road to continuous operations, one must come to the realization that applications must continuously dictate how the infrastructure performs.  Even the best written application will perform sporadically in a production environment if it doesn’t receive the proper allotment of infrastructure resources at any given time.  The reason this is so tricky today in the enterprise cloud is that applications share the infrastructure.  Not only are the needs of applications continuously changing, they run on infrastructure that house several other applications each of which has constantly changing needs.  This complexity is at the root of performance unreliability.
```
Comment (phrawzty) [inline site/content/events/2015-siliconvalley/proposals/Application Driven Infrastructure/index.txt]: Leading `G` - perhaps a typo?

File: site/content/events/2015-siliconvalley/proposals/Application Driven Infrastructure/index.txt [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+extension: html
+filter:
+- erb
+- markdown
+dirty: true
+proposal: true
+talk: true
+ignite: false
+selected: false
+author: Speaker 34
```
Comment (phrawzty) [inline site/content/events/2015-siliconvalley/proposals/Application Driven Infrastructure/index.txt]: Do the speakers not have names?

File: site/content/events/2015-siliconvalley/proposals/Application Driven Infrastructure/index.txt (hunk 1 above)
Comment (pmoosh) [inline site/content/events/2015-siliconvalley/proposals/Application Driven Infrastructure/index.txt]: Good catch - thanks

File: site/content/events/2015-siliconvalley/proposals/Application Driven Infrastructure/index.txt (hunk 2 above)
Comment (pmoosh) [inline site/content/events/2015-siliconvalley/proposals/Application Driven Infrastructure/index.txt]: Speaker selection is anonymous, which is why there is no name.

Comment (phrawzty) [inline site/content/events/2015-siliconvalley/proposals/Application Driven Infrastructure/index.txt]: Oh, neat! Great idea!

**Pull request merged**